veecle-telemetry-ui/src/ui/timeline/traces.rs

```diff
@@ -416,6 +416,19 @@ fn paint_record(
 
             info.painter.rect_filled(rect, RECT_ROUNDING, rect_color);
         }
+
+        // use outline for selected span
+        if is_selected {
+            let outline_color = Color32::from_rgb(255, 200, 50);
+            let outline_stroke = Stroke::new(2.0, outline_color);
```

Comment:
```suggestion
            let outline_stroke = Stroke::new(2.5, outline_color);
```
`2.5` seems to work decently well with the middle stroke